api/CampusService.proto

```diff
@@ -160,3 +213,56 @@ message GetTopNewsReply {
   google.protobuf.Timestamp from = 4;
   google.protobuf.Timestamp to = 5;
 }
+
+
+message Cafeteria {
+  string cafeteria = 1;
+}
+
+message DishInCafeteria {
+  string cafeteria = 1;
+  string tokenizedDish = 2;
+}
+
+message GetCafeteriaRatingReply {
+  repeated CafeteriaRatingGet rating = 1;
+  int32 averageRating = 2;
+}
+
+
+message GetDishRatingReply {
+  repeated DishRatingGet rating = 1;
+  int32 averageRating = 2;
+}
+
+
+message CafeteriaRatingGet{
+  int32 rating = 1;
+  int32 cafeteria = 2;
```

Comment:
The "cafeteria" should be the name of the corresponding cafeteria, I added this information to the attributes. Since two users can add a rating at exactly the same time, I am planning to use an additional id in the database. All fields form the grpc call are not used as ids in the sql table.